Sources/Apollo/GraphQLHTTPRequestError.swift

```diff
@@ -1,10 +1,13 @@
 /// An error which has occurred during the serialization of a request.
 public enum GraphQLHTTPRequestError: Error, LocalizedError {
+  case cancelledByDeveloper
```

Comment:
I think the enum case should be changed too, for the same reason the `errorDescription` did.